packages/infra/lib/patient-import-nested-stack.ts

```diff
@@ -42,9 +44,12 @@ function settings() {
       visibilityTimeout: Duration.seconds(patientCreateLambdaTimeout.toSeconds() * 2 + 1),
       createRetryLambda: false,
     },
-    waitTime,
+    waitTime: waitTimePatientCreate,
   };
-  const patientQueryLambdaTimeout = Duration.minutes(15).minus(Duration.seconds(5));
+  const waitTimePatientQuery = Duration.minutes(waitTimePatientQueryInSeconds);
+  const patientQueryLambdaTimeout = Duration.seconds(waitTimePatientQueryInSeconds + 30).minus(
+    Duration.seconds(5)
+  ); // 25secs for processinng
```

Comment:
for later: ditto simplify

Reply:
Will handle in v1.

Reply:
Done via patch https://github.com/metriport/metriport/pull/2862